src/VcrHandler.php

```diff
@@ -27,7 +27,7 @@ public static function turnOn($cassette)
             $handler->after('allow_redirects', new static($cassette));
             return $handler;
         } else {
-            $responses = json_decode(file_get_contents($cassette), true);
+            $responses = json_decode(utf8_decode(file_get_contents($cassette)), true);
```

Comment:
I worry that you're `utf8_decode`ing the entire JSON blob, rather than just the `body` as encoded below. I think you need to `json_decode()` then `utf8_decode()` just the `body` of each response.